website/content/commands/acl/set-agent-token.mdx

```diff
@@ -26,7 +26,7 @@ are not supported from commands, but may be from the corresponding HTTP endpoint
 
 ## Usage
 
-Usage: `consul acl set-agent-token [options] TYPE TOKEN`
+Usage: `consul acl set-agent-token [options] [TYPE](#token-types) TOKEN`
```

Comment:
Note to self: check whether this anchor link works as expected

Reply:
Yes, but it looks like this won't work. Was trying to make an explicit connection to the list of types below (rather than an implicit one).